### PR TITLE
docs: documentation changes for clarity

### PR DESCRIPTION
**Pull request by goodolLakshay**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [x] Documentation
- [ ] CI/CD

## Description
Made changes in the following files:

README.md
bug_report.yaml
feature_request.yml
PULL_REQUEST_TEMPLATE.md

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context
I came across the repo and read the documentation and found areas of improvement, made the changes, and created this PR.
This change is required to improve the clarity and readability of the project's documentation. It addresses no code-related issues.

## How did you test it?
I read the code and checked the corresponding preview.

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,39 +213,39 @@ and improve control.
 But integrating and maintaining multiple processors needs a lot of dev effort.
 Why should devs across companies repeat the same work?
 Why can't it be unified and reused? Hence, HyperSwitch was born to create that
-reusable core and let companies build and customize on top of it.
+reusable core and let companies build and customise it on top of it.
```
Comment (Narayanbhat166) [inline README.md:216]: The `it` here seems unnecessary

Comment (goodolLakshay) [reply README.md]: > The `it` here seems unnecessary

made it more clear as per the subject of the sentence.

**Review by Narayanbhat166 — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -35,8 +35,8 @@ Why is this change required? What problem does it solve?
 If it fixes an open issue, please link to the issue here.
 
 If you don't have an issue, we'd recommend starting with one first so the PR
-can focus on the implementation (unless its an obvious bug or documentation fix
-that will have little conversation).
+can focus on the implementation (unless it is an obvious bug or documentation fix
+that will have a little conversation).
```
Comment (SanchithHegde) [inline .github/PULL_REQUEST_TEMPLATE.md:39]: The intent here is: "please open an issue for this PR if one doesn't already exist, unless this is an obvious bug fix that requires little to no conversation."

So we can remove the `a` here, and keep it as either:

```suggestion
that will have little conversation).
```

or if we want to be verbose:

```suggestion
that will have little to no conversation).
```

File: README.md [hunk 3]
```diff
@@ -42,10 +42,10 @@ Using HyperSwitch, you can:
 - **Improve success rates** with seamless failover and auto-retries
 - **Reduce processing fees** with smart routing
 - **Customize payment flows** with full visibility and control
-- **Increase business reach** with local / alternate payment methods
+- **Increase business reach** with local/alternate payment methods
 
 > HyperSwitch is **wire-compatible** with top processors like Stripe, making it
-> easy to integrate.
+> Easy to integrate.
```
Comment (SanchithHegde) [inline README.md:48]: Nit: this need not be capitalized?

**Review by SanchithHegde — COMMENTED** [review]
Other than that, looks good to me! Thanks for the PR!

**Comment by goodolLakshay** [comment]
Any Updates?

**Comment by goodolLakshay** [comment]
> Any Updates?

@SanchithHegde @Narayanbhat166

**Review by SanchithHegde — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (SanchithHegde) [reply README.md]: Hey @goodolLakshay, I've updated the capitalization on the README. Previously, the rendered README looked like this, the 'E' being uppercase was clearly not preferable.

Screenshot of the rendered README

**Pull request merged**